spec/controllers/v1/post911_gi_bill_statuses_controller_spec.rb

```diff
@@ -54,8 +54,9 @@
 
       expect(response).to have_http_status(:not_found)
       json_response = JSON.parse(response.body)
-      expect(json_response['error']['title']).to eq('Not Found')
-      expect(json_response['error']['detail']).to eq('Icn not found.')
+      error = json_response['errors'][0]
```

Comment:
I see the key changes from "error" to "errors". Is it possible to have and return multiple errors? Wondering if that needs to be tested here and if the front end would need any changes for that?

Reply:
The FE data contract requests an errors list.  

https://department-of-veterans-affairs.github.io/va-digital-services-platform-docs/api-reference/#/benefits_status/getPost911GiBillStatus

In reality there will only ever be one error.  You can see this same pattern echoed in the [Lighthouse API docs](https://developer.va.gov/explore/api/education-benefits/docs?version=current).  I think it's just convention.